homework/index.js

```diff
@@ -27,21 +30,138 @@
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+  // fetch url contributor
+  function fetchContributorsUrl(contributor) {
+    const cont = document.getElementById('cont');
+    cont.innerHTML = '';
+    fetchJSON(contributor)
+      .then(myContributor => {
+        for (let j = 0; j < myContributor.length; j++) {
+          const nameOfContributor = myContributor[j].login;
+          const numberOfContributor = myContributor[j].contributions;
+          const avatarUrl = myContributor[j].avatar_url;
+          const avatarContainer1 = createAndAppend('div', cont, {
+            class: 'avatar_div',
+          });
+          const avatarContainer2 = createAndAppend('div', cont, { class: 'avatar_div data' });
+
+          createAndAppend('img', avatarContainer1, { src: avatarUrl });
+          createAndAppend('a', avatarContainer2, {
+            text: nameOfContributor.toUpperCase(),
+            href: `https://github.com/${nameOfContributor}`,
+          });
+          createAndAppend('br', avatarContainer2, {});
+
+          createAndAppend('span', avatarContainer2, {
+            text: 'Forks: ',
+          });
+          createAndAppend('span', avatarContainer2, {
+            text: numberOfContributor,
+            class: 'result label',
+          });
+          createAndAppend('br', avatarContainer2, {});
+        }
+      })
+      .catch(() => {
+        createAndAppend('div', cont, {
+          text: 'there is no contributors available',
+          class: 'error',
+        });
+      });
+  }
+
+  function reloadSelectedRepository(valueOfSelectedRepository, selectedData) {
+    const info = document.getElementById('info');
+    info.innerHTML = '';
+    const getDescription = selectedData[valueOfSelectedRepository].description;
+    const getForks = selectedData[valueOfSelectedRepository].forks;
+    const getUpdatedAt = selectedData[valueOfSelectedRepository].updated_at;
+    const dataName = selectedData[valueOfSelectedRepository].name;
+    const getContributorsUrl = selectedData[valueOfSelectedRepository].contributors_url;
+
+    if (getDescription == null) {
+      const noDescription = createAndAppend('li', info, {});
+      createAndAppend('span', noDescription, {
+        text: 'Description: there is no description',
+        class: '',
+      });
+    } else {
+      const liDescription = createAndAppend('li', info, {});
+      createAndAppend('span', liDescription, { text: 'Description: ' });
+      createAndAppend('span', liDescription, { text: getDescription, class: 'result' });
+    }
+
+    const link = createAndAppend('li', info, {});
+    createAndAppend('span', link, { text: 'link: ' });
+    createAndAppend('a', link, {
+      text: dataName,
+      target: '_blank',
+      href: `https://github.com/HackYourFuture/${info}`,
```

Comment:
The link to repo on GitHub is in `selectedData[valueOfSelectedRepository].html_url`, but if you still want to compose it; you need to replace `info` with `dataName`.